Sources/SwiftSyntaxBuilder/Tokens.swift.gyb

```diff
@@ -62,4 +62,9 @@ public extension TokenSyntax {
   static var eof: TokenSyntax {
     SyntaxFactory.makeToken(.eof, presence: .present)
   }
+  /// The `open` token
```

Comment:
One minor nitpick: Could you rephrase this to 
```suggestion
  /// The `open` contextual token
```

It should make it clear why it is not generated using `gyb`.